.github/workflows/build.yaml

```diff
@@ -68,6 +68,54 @@ jobs:
       - name: Run bats tests
         run: make e2e-bats
 
+  test-sarif:
+    needs: build-and-test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Download executable
+        uses: actions/download-artifact@v2
+        with:
+          name: bin
+      
+      - name: Set permissions to file
```

Comment:
nit: Maybe we can merge both together into a single step, i.e. `Make kube-linter executable and verify version`?